spinalcordtoolbox/scripts/sct_compute_compression.py

```diff
@@ -0,0 +1,644 @@
+from spinalcordtoolbox.image import Image
+from spinalcordtoolbox.centerline.core import get_centerline, ParamCenterline
+from spinalcordtoolbox.types import Centerline
+from spinalcordtoolbox import __data_dir__
+from spinalcordtoolbox.scripts import sct_process_segmentation
+
+
+logger = logging.getLogger(__name__)
+
+
+NEAR_ZERO_THRESHOLD = 1e-6
+
+
+# PARSER
+# ==========================================================================================
+def get_parser():
+    parser = SCTArgumentParser(
+        description='Compute normalized morphometric measures to assess spinal cord compression.\n'
+                    '\n'
+                    'Metrics are normalized using the non-compressed levels above and below the compression site using '
+                    'the following equation:\n'
+                    '\n'
+                    '    ratio = (1 - mi/((ma+mb)/2))\n'
+                    '\n'
+                    'Where mi: metric at the compression level, ma: metric above the compression level, mb: metric '
+                    'below the compression level.\n'
+                    '\n'
+                    'Additionally, if the "-normalize" flag is used, metrics are normalized using a database built '
+                    'from healthy control subjects. This database uses the PAM50 template as an anatomical reference '
+                    'system.\n'
+                    '\n'
+                    'Reference: Miyanji F, Furlan JC, Aarabi B, Arnold PM, Fehlings MG. Acute cervical traumatic '
+                    'spinal cord injury: MR imaging findings correlated with neurologic outcome--prospective study '
+                    'with 100 consecutive patients. Radiology 2007;243(3):820-827.'
+    )
+
+    mandatoryArguments = parser.add_argument_group("\nMANDATORY ARGUMENTS")
+    mandatoryArguments.add_argument(
+        '-i',
+        metavar=Metavar.file,
+        required=True,
+        help='NIfTI file of spinal cord segmentation to compute morphometrics. If no normalization is wanted (if -i-PAM50 is not specified),'
+             ' metric ratio will take average along the centerline.'
+    )
+    mandatoryArguments.add_argument(
+        '-vertfile',
+        metavar=Metavar.file,
+        required=True,
+        help='Vertebral labeling file.'
+             'The input and the vertebral labelling file must be in the same voxel coordinate system'
+             'and must match the dimensions between each other.'
+    )
+    mandatoryArguments.add_argument(
+        '-l',
+        metavar=Metavar.file,
+        required=True,
+        help='NIfTI file that includes labels at the compression sites. Each compression site is denoted by a single voxel of value `1`. '
+             'The input and the compression label file must be in the same voxel coordinate system'
+             'and must match the dimensions between each other.'
+             'Example: sub-001_T2w_compression_labels.nii.gz'
```

Comment:
Just reorganize to match the previous arguments.

```suggestion
        help='NIfTI file that includes labels at the compression sites. Each compression site is denoted by a single voxel of value `1`.  Example: sub-001_T2w_compression_labels.nii.gz'
             'Note: The input and the compression label file must be in the same voxel coordinate system'
             'and must match the dimensions between each other.'
```